src/middlewared/middlewared/migration/0006_system_cert.py

```diff
@@ -0,0 +1,8 @@
+async def migrate(middleware):
+    # This is required because earlier middleware was running considering openssl security level to be 1
+    # but with moving to debian, the default now is 2 which enforces security standards. It means that potentially
+    # user might have configured cert for system which might not comply with security level 2 and UI becoming
+    # inaccessible in this regard because of that
+    system_cert = (await middleware.call('system.general.config'))['ui_certificate']
```

Comment:
Do we always have this certificate at this point? I recall seeing systems without it.

Reply:
We would have it as the crypto setup function ensures that. However i can still do this safely.